src/Ilios/CliBundle/Command/InstallUserZeroCommand.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Ilios\CliBundle\Command;
+
+use Ilios\CliBundle\Form\InstallUserZeroType;
+use Ilios\CoreBundle\Entity\Manager\AuthenticationManagerInterface;
+use Ilios\CoreBundle\Entity\Manager\SchoolManagerInterface;
+use Ilios\CoreBundle\Entity\Manager\UserManagerInterface;
+use Ilios\CoreBundle\Entity\Manager\UserRoleManagerInterface;
+
+use Matthias\SymfonyConsoleForm\Console\Helper\FormHelper;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Creates a first ("user zero") account with Course Director privileges.
+ *
+ * Class InstallUserZeroCommand
+ * @package Ilios\CoreBundle\Command
+ */
+class InstallUserZeroCommand extends ContainerAwareCommand
+{
+    /**
+     * @var string
+     */
+    const USERNAME = 'zero_user';
+
+    /**
+     * @var string
+     */
+    const PASSWORD = 'Ch4nge_m3';
+
+    /**
+     * @var string
+     */
+    const FIRST_NAME = 'User';
+
+    /**
+     * @var string
+     */
+    const LAST_NAME = 'Zero';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('form:install_user_zero')
```

Comment:
I think this should be name spaces under ilios and made less explicit.
Maybe ilios:setup:first-user

Then we might have other ilios:setup tasks in the same space.
It looks like most other console commands are dasherized instead of underscored.